source/examples/rapids-morpheus-pipeline/notebook.md

```diff
@@ -0,0 +1,164 @@
+  check the original example on the [Morpheus documentation](https://docs.nvidia.com/morpheus/examples/nlp_si_detection/readme.html#background)
+
+- **cuDF Network Traffic Analysis Stage**: We incorporate an additional analysis stage using `cuDF` to perform
+  some network traffic analytics for enriched context and anomaly detection. [Stage code here](https://github.com/rapidsai/deployment/source/examples/rapids-morpheus-pipeline/scripts/pipeline-dockerfile/network_traffic_analyzer_stage.py)
+
+- **Kafka Output Sink**: Finally, the processed and enriched data, with SI detection results
+  and traffic insights, is published to a downstream Kafka topic for further processing, alerting,
+  or storage.
+
+The entire pipeline is containerized and deployed on **Amazon EKS**, leveraging Kubernetes
+for orchestration, scalability, and resiliency in a cloud-native environment.
+
+## Deployment Components
+
+The pipeline is deployed on Amazon EKS using several Kubernetes manifests:
+
+### Kafka Deployment (`k8s/kafka`)
+
+The Kafka cluster is deployed using the [Strimzi Operator](https://strimzi.io/), which simplifies Kafka deployment and
+management on Kubernetes. See instructions on section [Deploying on EKS](deploying-on-eks)
+
+The deployment configuration includes:
+
+- Kafka cluster setup `kafka-single-node.yaml`
+
+  This is a modification of the file [https://strimzi.io/examples/latest/kafka/kafka-single-node.yaml](https://strimzi.io/examples/latest/kafka/kafka-single-node.yaml) where we modify:
+
+  - Cluster name to `kafka-cluster`
+  - Modify the volume to use `type: ephemeral` and use `sizeLimit: 5Gi` (instead of `size: 100Gi` that corresponded to
+    `type: persistent-claim`)
+
+- Kafka topics setup
+- Kafka UI
+
+### Kafka Producer Deployment (`k8s/kafka-producer`)
+
+The Kafka producer is deployed as a separate pod using the `kafka-producer.yaml` manifest. It continuously generates
+and publishes network data to the Kafka topic. This producer script is containerized using a custom Docker image that can be built using the Dockerfile in the `scripts/producer-dockerfile` directory.
+
+- Uses `kafka-python` for message production.
+- Contains the producer script for generating network data.
+
+### Triton-Morpheus Deployment (`k8s/triton`)
+
+The inference server is deployed using the NVIDIA Morpheus- Triton Inference Server docker image
+`nvcr.io/nvidia/morpheus/morpheus-tritonserver-models:25.02`.
+
+### Morpheus Pipeline Deployment (`k8s/morpheus-pipeline`)
+
+The core processing pipeline is deployed as a separate pod that, uses an image that can be built using the Dockerfile in
+the `pipeline-dockerfile` directory.
+
+- Runs the Morpheus nightly 25.06 conda build
+- Contains all pipeline and stage scripts `scripts/pipeline-dockerfile/*.py`
+- Processes the streaming data through the various stages
+
+(deploying-on-eks)=
+
+## Deploying on EKS
```

Comment:
We should probably `docref` the EKS page in here somewhere

Reply:
I though about it. I'm not sure how though because the EKS page is a simple rapids example.